jboss/container/wildfly/launch/elytron/added/launch/elytron.sh

```diff
@@ -329,6 +335,37 @@ configure_security_domains() {
     configure_http_application_security_domains
     configure_ejb_application_security_domains
   fi
+  if [ -n "${ELYTRON_SECDOMAIN_NAME}" ]; then
+    validate_elytron_security_domains_env
+    if [ -z "${ELYTRON_SECDOMAIN_CORE_REALM}" ]; then
+      elytron_sec_domain="${ELYTRON_SECDOMAIN_NAME}"
+      configure_elytron_realm application-properties-"${ELYTRON_SECDOMAIN_NAME}"
+      configure_elytron_security_domain "${ELYTRON_SECDOMAIN_NAME}" application-properties-"${ELYTRON_SECDOMAIN_NAME}"
```

Comment:
Suggestion: quote all the argument to avoid problems if there are white spaces:
instead of application-properties-"${ELYTRON_SECDOMAIN_NAME}", use "application-properties-${ELYTRON_SECDOMAIN_NAME}"